dependencies/pom.xml

```diff
@@ -81,6 +82,10 @@
         <zstd.version>1.5.0-4</zstd.version>
         <xstream.version>1.4.20</xstream.version>
 
+        <!-- The `httpcore` and `httpclient` have been removed in spring-boot 3.1.0 and above. -->
+        <httpcore.version>4.4.16</httpcore.version>
```

Comment:
Is it the same as the implicit dependency version?

Reply:
Different versions

Reply:
It is the same version as in the later version of Spring Boot.
I tested the lower version of Spring Boot and found no issues.